### DISCUSSION TITLE
deploy to directories in domain

### DISCUSSION THREAD

**Discussion by doncqueurs** [basecamp/kamal#1241] [Q&A]
Hi, 
I believe Kamal could be a game changer for what I am doing building small apps. It looks very interesting, but I've got one question. I have a number of small apps that I am currently hosting under the same subdomain like sub.domain.com/app1, sub.domain.com/app2 etc. For each app, I have a separate git repo and I would like to be able to deploy changes independent from each other. Is there a way to deploy to a directory (under a subdomain) (thus creating a separate container for each app) while using the kamal proxy to manage everything on the domain and the certificate etc. 
Jochem

**Comment by tuladhar** [comment]
Currently, Kamal only supports virtual hosting of multiple apps with different domain names (such as app1.example.com, app2.example.com). It does not support path-based virtual hosting, like you mentioned.

Since, Kamal automatically handles SSL certificate for the subdomains, any reason stopping you from using different subdomains for your apps?

**Reply by doncqueurs** [reply]
Thanks for the clarification. It confirms the impression I already had. 

The reason for the desire for path-based virtual hosting is that I host multiple apps for the same customer under a subdomain.
